tests/test_healpixRoi.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+from hawc_hal import HealpixMapROI, HAL
+from hawc_hal.maptree import map_tree_factory
+from threeML import Model
+from astromodels import PointSource, ExtendedSource, Powerlaw, Gaussian_on_sphere
+import healpy as hp
+import numpy as np
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+import os
+
+from conftest import check_map_trees
+
+
+def Sky2Vec( ra, dec ):
+  c = SkyCoord( frame = "icrs", ra=ra*u.degree, dec=dec*u.degree )
+  theta = (90.0*u.degree-c.dec).to(u.radian).value
+  phi = c.ra.to(u.radian).value
+  vec = hp.pixelfunc.ang2vec(theta, phi)
+  return vec
```

Comment:
Just a suggestion, but woulnd't 

```
def Sky2Vec(ra, dec):
    return hp.pixelfunc.ang2vec(ra, dec, lonlat=True )
```
 do the same thing?

Reply:
Mmm, probably. I copied that function from another test.